source/docs/casper/developers/writing-onchain-code/simple-contract.md

```diff
@@ -246,6 +298,7 @@ This step adds the individual entry points to a `counter_entry_points` object us
         EntryPointType::Contract,
     ));
```

Comment:
I think we should add the '}' at the end as well. This way user can really just copy-paste the snippet

```suggestion
    ));
}
```